packages/api/tests/entities/Assignment.test.ts

```diff
@@ -1,16 +1,36 @@
 import { TestClient } from '../utils/TestClient';
+import { PathInput } from '../../types';
+import { ModuleType } from '../../src/Module/Module.entity';
 import * as random from '../../src/Database/seeders/random';
 
 beforeAll(async () => { await TestClient.start(); });
 afterAll(async () => { await TestClient.stop(); });
 
+export const assignment1 = "";
```

Comment:
I think you don't need this

Reply:
The issue comes when I declare assignment1 inside the setup method it comes back as undefined.

Reply:
If I don't export the constant at the start it comes back as undefined when referenced, even though it's being assigned in the setup.